src/types/sourceTargetTypes.ts

```diff
@@ -60,21 +73,15 @@ export enum ChainDetails {
   SOURCE = 'sourceChainDetails',
   TARGET = 'targetChainDetails'
 }
-export interface SourceState {
```

Comment:
After some time it was no point having specific objects named per source / target like `sourceApiConnection ` or `targetConfigs` so i unified everything in one common type.